packetnetworking/distros/debian/test_bonded.py

```diff
@@ -94,17 +101,7 @@ def test_public_bonded_task_etc_network_interfaces(
             post-down route del -net 10.0.0.0/8 gw {ipv4priv.gateway}
         """
 
-    result = dedent(result).format(
-        ipv4pub=builder.ipv4pub.first,
-        ipv6pub=builder.ipv6pub.first,
-        ipv4priv=builder.ipv4priv.first,
-        iface0=builder.network.interfaces[0],
-        iface1=builder.network.interfaces[1],
-        bonding_mode=builder.network.bonding.mode,
-        dns1=builder.network.resolvers[0],
-        dns2=builder.network.resolvers[1],
-    )
-    assert tasks["etc/network/interfaces"] == result
+    assert tasks["etc/network/interfaces"] == dedent(result)
```

Comment:
Not the point of this change

Reply:
I think you're mis reading the diff here. We were previously dedenting at `dedent(result).format(...)`, we are no longer calling `.format` so the dedent needs to go somewhere. Its here at the end because future commits are going to break up the one-large strings to be able to work in logic for variable number of nics and its not great to do within a function call.